Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Set default values for build arguments
-ARG DEFRA_VERSION=1.1.1
-ARG BASE_VERSION=11-jdk-jammy
+ARG DEFRA_VERSION=1.1.2
+ARG BASE_VERSION=17-jre-jammy
```

Comment:
should this be sdk instead?

Reply:
choose JRE instead of JDK - no development tools on the image, just the ones needed for execution

Reply:
@arunaukumar Be careful, I'm reasonably certain that at least one of your consumers requires the JDK for their execution images.

That said, this should be a safe (if unnecessary change) as all images will be produced still due to the image-matrix.json shenanigans which overrides the argument. The value in the Dockerfile is just a default for local development of the repository.